src/CFamily.UnitTests/Rules/RulesLoaderTest.cs

```diff
@@ -36,7 +36,8 @@ public class RulesLoaderTest
         public void Read_Rules()
         {
             var rulesLoader = CreateTestSubject();
-            rulesLoader.ReadRulesList().Should().HaveCount(410);
+            // 410 in RulesList.json + 2 in MisraRulesList.json
+            rulesLoader.ReadRulesList().Should().HaveCount(410 + 2);
```

Comment:
Added another dummy `MisraRulesList.json` file for this tests, which lists two rules (see next file).